### DISCUSSION TITLE
tp and tpa cooldowns

### DISCUSSION THREAD

**Discussion by elaineandsparky** [EssentialsX/Essentials#4585] [Ideas and Feature Suggestions] [answered]
Seperate tp and tpa cooldowns in the configs so they can be different rather than just this line:

The delay, in seconds, before a user actually teleports. If the user moves or gets attacked in this timeframe, the teleport is cancelled.
teleport-delay: 3

we have a 3 second delay so players cant just escape mobs but if a player is griefing for example and doesn't stop moving staff with /tphere perms can't tp a player away as the tp cancels when they move. We need that to be instant tp. Only way to stop someone at the moment is by kicking or banning them when no cooldown on /tphere would be enough.

**Comment by hirata-estate** [comment, marked as answer]
Already suggested in #4575.
